frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestFeatureToggleChange/StrategyChange.tsx

```diff
@@ -1,27 +1,59 @@
-import { Box, Typography } from '@mui/material';
+import { Box, Link, Typography } from '@mui/material';
 import { FC } from 'react';
 
-export const StrategyAddedChange: FC = ({ children }) => {
+interface IStrategyChangeProps {
+    onDiscard: () => void;
+}
+
+export const StrategyAddedChange: FC<IStrategyChangeProps> = ({
+    children,
+    onDiscard,
+}) => {
     return (
-        <Box sx={{ p: 1, display: 'flex', gap: 1 }}>
-            <Typography sx={theme => ({ color: theme.palette.success.main })}>
-                + Strategy Added:
-            </Typography>
-            {children}
+        <Box sx={{ p: 1, display: 'flex', justifyContent: 'space-between' }}>
+            <Box sx={{ display: 'flex', gap: 1 }}>
+                <Typography
+                    sx={theme => ({ color: theme.palette.success.main })}
+                >
+                    + Adding strategy:
+                </Typography>
+                {children}
+            </Box>
+            <Box>
+                <Link onClick={onDiscard}>Discard</Link>
+            </Box>
         </Box>
     );
 };
 
-export const StrategyEditedChange: FC = () => {
-    return <Box sx={{ p: 1 }}>Strategy Edited</Box>;
+export const StrategyEditedChange: FC<IStrategyChangeProps> = ({
+    children,
+    onDiscard,
+}) => {
+    return (
+        <Box sx={{ p: 1, display: 'flex', justifyContent: 'space-between' }}>
+            <Box sx={{ display: 'flex', gap: 1 }}>
+                <Typography>Editing strategy:</Typography>
+                {children}
+            </Box>
+            <Box>
+                <Link onClick={onDiscard}>Discard</Link>
+            </Box>
+        </Box>
+    );
 };
 
-export const StrategyDeletedChange: FC = () => {
+export const StrategyDeletedChange: FC<IStrategyChangeProps> = ({
+    onDiscard,
+}) => {
     return (
-        <Box sx={{ p: 1 }}>
+        <Box sx={{ p: 1, display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
Could be a styled component. Also, should we discard `p` in favor of `padding` ?

Reply:
Sure this was WIP and already switched to Styled components